android/src/main/java/com/amplitude/android/internal/gestures/AutocaptureWindowCallback.kt

```diff
@@ -33,6 +33,13 @@ internal class AutocaptureWindowCallback(
         return super.dispatchTouchEvent(event)
     }
 
+    private fun handleTouchEvent(event: MotionEvent) {
+        gestureDetector.onTouchEvent(event)
+        if (event.actionMasked == MotionEvent.ACTION_UP) {
+            gestureListener.onUp(event)
+        }
```

Comment:
Can this be handled in `gestureDector.onTouchEvent()` instead?

Reply:
Since GestureDetector is a built-in class and gestureDector is an instance of that, I don't think there is a way to add this logic to `gestureDector.onTouchEvent()` instead.

Reply:
Ah sorry I misread that these were both `gestureDetector` vs `gestureListener`